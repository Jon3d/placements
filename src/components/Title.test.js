import renderer from 'react-test-renderer';
import Title from './Title';

it('renders Title correctly', () => {
  const tree = renderer
    .create(<Title />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});