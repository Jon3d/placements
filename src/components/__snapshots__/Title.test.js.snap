// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Title correctly 1`] = `
<h2
  className="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom css-gi7hi0-MuiTypography-root"
/>
`;
